src/index.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="./@types/ethjs-util/index.ts"/>
```

Comment:
I think you can just do an `import "./@types/ethjs-util/index.ts";` here. I'm not sure if this triple-slash reference would work after compilation, as `index.ts` won't exist in the build.

Reply:
@alcuadrado 

Yeah this is weird but `tsc` auto-generates the contents the @types folder in the `dist` creating a `.d.ts` file. 
```
dist/
  @types/
    ethjs-util/
      index.d.ts
      index.js
      index.js.map
```
*And* it rewrites the triple slash reference to point to the `.d.ts` in the dist's main `index.d.ts`.

Reply:
🤯